### DISCUSSION TITLE
科學家用 Jetson Nano 開發 AI 雷射砲台，為了準確擊殺蟑螂

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#629] [General]
科技媒體 ExtremeTech 報導，愛丁堡赫瑞瓦特大學研究人員 Ildar Rakhmatulin 嘗試將有 AI 的機器視覺系統用於家庭害蟲防治。開始他以蚊子為主要研究目標，也成功製作擊殺蚊子的雷射裝置；下一個目標便是蟑螂，畢竟眾所皆知，蟑螂對公共衛生有更大影響。
此 AI 雷射砲台是以 NVIDIA「Jetson Nano」為核心，Rakhmatulin 和團隊使用 1 千張不同照明下的蟑螂圖片，訓練 Nano 辨識目標並追蹤蟑螂行動。

我的想法：雖然這個酷東西我一定要搞到手，但如內文所說，如果沒有適當防護措施，雷射仍會傷害人眼。所以其實也是充滿著危險！
它的辨識方法很酷，我也蠻想看那1000張不同照明下的蟑螂圖片，說不定十分精彩呢！

https://ccc.technews.tw/2022/10/03/jetson-nano/

**Comment by magicmicheal** [comment]
感覺市面上許多的殺蟑商品的效率比這項功能高，價錢也比較便宜，不知道在暗處這項產絣如何發揮功用

**Comment by awehao** [comment]
感覺還是會對人體造成傷害，不過確實我也想要有一個，希望可以背著當一個行動殺蟑機，晚上孤身一人走在路上常常會被水溝跑出來的蟑螂嚇到，真的很煩，若是這個東西可以普及化，那麼想必會造福世界

**Comment by RogelioKG** [comment]
不知道有沒有人能用 AI 做一個雷射砲塔擊殺喇牙，
我知道喇牙是益蟲，可是為什麼他們長的那麼恐怖呢🥲，
而且中央未免也太多喇牙了吧，
聽說喇牙愛吃蟑螂，可我並沒有感覺中央蟑螂很多R，
這學期已經有兩次喇牙闖入紀錄了。
上次喇牙小可愛出來逛街時，
室友看著我，一臉「這間宿舍已經不能住人了」的強忍鎮定表情。
好無奈啊。

**Comment by awkward-willy** [comment]
文中有提到雷射可能會傷害人眼，所以這項產品的實際使用情況可能還有待觀察，不過這樣的科技確實能大幅改變人們的生活，畢竟蚊子、蟑螂等除了造成衛生問題外，還可能散播疾病，若在未來能精準消滅，肯定是一大福音之一。

**Comment by manipulateed** [comment]
雷射問題肯定可以先去偵測有無人來的蹤跡，再去決定要不要開啟，蟑螂的行蹤飄忽不定，能在人沒有的視野下擊殺他們，很有效率的展現更是能提高生活品質。

**Comment by larry40305** [comment]
或許改變殺蟑螂的工具能提升這項產品的實用度及安全性。
